components/FileUploader.js

```diff
@@ -210,11 +221,15 @@ class FileUploader extends Component {
                   <span className="bx--file__state-container">
                     <Filename
                       status={filenameStatus}
-                      onClick={evt => this.handleClick(evt, index)}
+                      onClick={evt => {
+                        if (filenameStatus === 'edit') {
```

Comment:
__Changes:__ Add conditional so that `onClick` is only fired when `filenameStatus === 'edit'`.